config/config.mainnet.env

```diff
@@ -7,3 +7,8 @@ MCUSD_ADDRESS=0x918146359264c492bd6934071c6bd31c854edbc3
 MCEUR_ADDRESS=0xe273ad7ee11dcfaa87383ad5977ee1504ac07568
 MCREAL_ADDRESS=0x9802d866fde4563d088a6619f7cef82c0b991a55
 
+INVITES_POLLING_INTERVAL=3600000
+ACCOUNTS_POLLING_INTERVAL=3600000
+ATTESTATIONS_POLLING_INTERVAL=3600000
```

Comment:
hourly. we probly don't need these at all, but I want to double check that before turning them off